spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,23 @@
+require 'beaker-rspec'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
```

Comment:
there are so many different ways to install puppet in a module, is this the current best practice?